internal/tekton-bundle/tekton-bundle_test.go

```diff
@@ -0,0 +1,49 @@
+package tekton_bundle
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	numberOfPipelines  = 2
+	numberOfConfigMaps = 2
+)
+
+var _ = Describe("Tekton bundle", func() {
+
+	It("should return correct pipeline folder path on okd", func() {
+		path := getPipelineBundlePath(true)
+		Expect(path).To(Equal("/data/tekton-pipelines/okd/"))
+	})
+
+	It("should return correct pipeline folder path on kubernetes", func() {
+		path := getPipelineBundlePath(false)
+		Expect(path).To(Equal("/data/tekton-pipelines/kubernetes/"))
+	})
+
+	It("should load correct files and convert them", func() {
+		path, _ := os.Getwd()
+
+		pipelinePath := filepath.Join(path, "test-bundle-files/test-pipelines/")
+
+		pipelineFiles, err := readFolder(pipelinePath)
+		Expect(err).ToNot(HaveOccurred())
+		files := [][]byte{}
+		files = append(files, pipelineFiles...)
```

Comment:
Hi, `files` variable is needed because the type of `taskFile` is `[]byte` and the type of `pipelineFiles` is `[][]byte` and I'm not sure how we can not use it.

Reply:
I think I posted this review comment on the commit that added `pipelineFiles`, but not `taskFile`.
Now it is irrelevant when the PR is merged.